CONTRIBUTING.md

```diff
@@ -51,10 +50,10 @@ You may be able to find others to collaborate with for mutual benefit.
 
 ## Create the working (topic) branch
 
-Create a "topic" branch on which you will work.  The convention is to name the branch using the JIRA issue key.  If there is not already a JIRA issue covering the work you want to do, create one.  Assuming you will be working from the master branch and working on the JIRA issue `UP-123`:
+Create a "topic" branch on which you will work.  The convention is to name the branch using the GitHub issue number.  If there is not already a GitHub issue covering the work you want to do, create one.  Assuming you will be working from the master branch and working on the GitHub issue `#42`:
```

Comment:
Once there's a topic branch, make a WIP Pull Request and skip the redundant GitHub issue. Pull Requests are issues except better. Issues are a fallback for when a contributor is prepared to articulate an issue but not yet prepared to create a topic branch and start working the issue. Issues for things that could already have a Pull Request will add more noise than signal.

I feel passionately about this point, but don't need to hash it out here. When I get to it I'll propose an edit to `CONTRIBUTING.md` and then we can engage with a concrete proposed change rather than just with philosophy. :)